EncryptionTests/Encryption Tests/CDTQIndexManagerEncryptionTests.m

```diff
@@ -209,4 +210,21 @@ - (void)testCreateQueryIndexManagerWithFixedKeyProviderFailsIfDBIsEncryptedButKe
     XCTAssertNotNil(err, @"There is an error");
 }
 
+- (void)testTextSearchIsAvailable
+{
+    CDTHelperFixedKeyProvider *provider = [CDTHelperFixedKeyProvider provider];
+    CDTDatastore *datastore = [self.factory datastoreNamed:@"is_fts_activated"
+                                 withEncryptionKeyProvider:provider
+                                                     error:nil];
+    
+    CDTQIndexManager *im = [[CDTQIndexManager alloc] initUsingDatastore:datastore error:nil];
+
+    NSSet *compileOptions = [DBQueryUtils compileOptions:im.database];
+    NSSet *ftsCompileOptions = [NSSet setWithArray:@[ @"ENABLE_FTS3", @"ENABLE_FTS3_PARENTHESIS" ]];
```

Comment:
I think the check for `ENABLE_FTS3` and `ENABLE_FTS3_PARENTHESIS` is too restrictive. It is only necessary to confirm that the `ENABLE_FTS3` compile option is available to ensure that text search will work.  That is all that we check for with `isTextSearchEnabled` in the index manager.  `ENABLE_FTS3_PARENTHESIS` is a SQLite compile option that will allow you to use enhanced query syntax along with FTS but is not necessary for text search itself to function.